src/dateTimePicker.test.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import TestUtils from 'react-addons-test-utils';
+import DateTimePicker from './DateTimePicker';
+import CalendarComponent from './CalendarComponent';
+import TimeInput from './TimeInput';
+
+fdescribe('DateTimePicker', function() {
+
+	let dateTimeComponent,
+		calendarComponent,
+		timeComponent;
+		// dateTimeLocalComponent;
+
+	const year = 2018,
+		month = 0,
+		day = 1,
+		dateStr = `${year}-0${month+1}-0${day}`,
+		hours = 23,
+		minutes = 11,
+		datetime = new Date(Date.UTC(year, month, day, hours, minutes));
+
+	// datetime.setHours(hours, minutes);
+
+	beforeEach(() => {
+		dateTimeComponent = TestUtils.renderIntoDocument(
+			<DateTimePicker name='start_time'
+				value={datetime}
+				forceCalendar />);
+		calendarComponent = TestUtils.findRenderedComponentWithType(dateTimeComponent, CalendarComponent);
+		timeComponent = TestUtils.findRenderedComponentWithType(dateTimeComponent, TimeInput);
+	});
+
+	afterEach(() => {
+		dateTimeComponent = null;
+		timeComponent = null;
+		calendarComponent = null;
+	});
+
+	it('exists', function() {
+		dateTimeComponent = TestUtils.renderIntoDocument(<DateTimePicker name='start_time' forceCalendar />);
+		expect(() => TestUtils.findRenderedComponentWithType(dateTimeComponent, CalendarComponent)).not.toThrow();
+		expect(() => TestUtils.findRenderedComponentWithType(dateTimeComponent, TimeInput)).not.toThrow();
+	});
+
+	it('values for date and time are set in state', function() {
+		expect(dateTimeComponent.state.datetime).toEqual(datetime);
+	});
+
+	it('accepts a Date as a value', function() {
+		dateTimeComponent = TestUtils.renderIntoDocument(
+			<DateTimePicker name='start_time'
+				value={new Date(dateStr)}
+				forceCalendar />);
+		expect(dateTimeComponent.state.datetime).toEqual(new Date(dateStr));
+	});
+
+	it('accepts a datetime object as value', function() {
```

Comment:
seems like a duplicate of https://github.com/meetup/meetup-web-components/pull/88/files#diff-1da4556b5f5f8d13c410ac257b8e8281R45